proposals/description/fip_99.ts

```diff
@@ -0,0 +1,53 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_99: ProposalDescription = {
+  title: 'FIP-99: Sell RAI',
+  commands: [
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address, address, uint256)',
+      arguments: ['{rariPool9RaiPCVDeposit}', '{aaveRaiPCVDeposit}', '8000'],
+      description: 'Withdraw all Rai from the rari pool 9 pcv deposit and sent to the aave rai pcv deposit'
+    },
+    {
+      target: 'aaveRaiPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: [],
+      description: 'Call deposit() on the aave rai pcv deposit'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{raiPriceBoundPSM}'],
+      description: 'Grant the MINTER_ROLE to the raiPriceBoundPSM'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantPCVController(address)',
+      arguments: ['{raiPCVDripController}'],
+      description: 'Grant the PCV_CONTROLLER_ROLE to the rai pcv drip controller'
+    },
+    {
+      target: 'raiPriceBoundPSM',
+      values: '0',
+      method: 'pauseMint()',
+      arguments: [],
+      description: 'Pause mint on the rai price-bound psm'
+    },
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'setSafeAddresses(address[])',
+      arguments: [['{rariPool9RaiPCVDeposit}', '{aaveRaiPCVDeposit}', '{raiPriceBoundPSM}']],
```

Comment:
You probably don't want to treat the PriceBoundNonCustodial PSM as a safe address. It really shouldn't be holding onto funds other than received FEI

Reply:
It's not non-custodial